drivers/util.py

```diff
@@ -753,6 +772,12 @@ def get_slaves_attached_on(session, vdi_uuids):
     master_ref = get_this_host_ref(session)
     return [x for x in host_refs if x != master_ref]
 
+def get_enabled_hosts(session):
+    """
+    Returns a list of host refs that are enabled in the pool.
+    """
+    enabled_hosts = [host[0] for host in session.xenapi.host.get_all_records_where('field "enabled" = "true"').items()]
```

Comment:
```suggestion
    enabled_hosts = list(session.xenapi.host.get_all_records_where('field "enabled" = "true"').keys())
```
Wouldn't that do the trick just fine?

Reply:
Pythonic! Done.

Reply:
if only refs are needed, `session.xenapi.host.get_all_where('field "enabled" = "true"')` is available, no need to get the records

Reply:
> if only refs are needed, `session.xenapi.host.get_all_where('field "enabled" = "true"')` is available, no need to get the records

I'm getting `XenAPI.Failure: ['MESSAGE_METHOD_UNKNOWN', 'host.get_all_where']`, is it due to my pending updates?

Reply:
Hmm, are you running 8.2? Apparently this was only added in March 2024, so should probably keep the get_all_records_where for now, could just leave a note to upgrade it in the future (and for the neighboring functions too)

Reply:
Its `8.3.0` with `xapi-core-24.19.2-1.9.xcpng8.3.x86_64`